homeassistant/components/isy994.py

```diff
@@ -132,6 +134,20 @@ def _categorize_programs() -> None:
                         PROGRAMS[component].append(program)
 
 
+def _categorize_weather() -> None:
+    """Categorize the ISY994 weather data."""
+    global WEATHER_NODES
```

Comment:
I was actually considering removing the event decorators because `partial` does not work with async. I also think that you're the only one using them ;-)